src/dfeed/loc/turkish.d

```diff
@@ -763,145 +765,147 @@ string translate(string s)
 		case `Settings saved.`:
 			return `Ayarlar kaydedildi.`;
 		case `Warning: cookie size approaching RFC 2109 limit.`:
-			return `Uyarı: tanımlama bilgisi boyutu RFC 2109 sınırına yaklaşıyor.`;
+			return `Uyarı: çerez boyutu RFC 2109 sınırına yaklaşıyor.`;
 		case `Please consider %screating an account%s to avoid losing your read post history.`:
-			return `Okuma gönderi geçmişinizi kaybetmemek için lütfen %s hesabınızı %s göstermeyi düşünün.`;
+			return `Okuma geçmişinizi kaybetmemek için lütfen %shesap açtırın%s.`;
 		case `Forums`:
-			return `Forumlar`;
+			return `Forumlarda`;
 		case `%s group`:
-			return `%s grubu`;
+			return `%s grubunda`;
 		case `View subscription:`:
-			return `Aboneliği görüntüle:`;
+			return ``;
 		case `It looks like there's nothing here! No posts matched this subscription so far.`:
-			return `Görünüşe göre burada hiçbir şey yok! Şu ana kadar bu abonelikle eşleşen gönderi yok.`;
+			return `Görünüşe göre burada hiçbir şey yok! Henüz bu abonelikle eşleşen gönderi yok.`;
 		case `This subscription has been deactivated.`:
 			return `Bu abonelik devre dışı bırakıldı.`;
 		case `If you did not intend to do this, you can reactivate the subscription's actions on your %ssettings page%s.`:
-			return `Bunu yapmak istemediyseniz, aboneliğin eylemlerini %ssettings sayfanızda %s yeniden etkinleştirebilirsiniz.`;
+			return `Bunu yapmak istemediyseniz, aboneliğin eylemlerini %sayarlar sayfanızda%s yeniden etkinleştirebilirsiniz.`;
 		case `Hint`:
 			return `İpucu`;
 		case "Is the CAPTCHA too hard?\nRefresh the page to get a different question,\nor ask in the %s#d IRC channel on Freenode%s.":
-			return `CAPTCHA çok mu zor?\nFarklı bir soru almak için sayfayı yenileyin\nveya %sFreenode üzerindeki #d IRC kanalında%s sorun.`;
+			return `CAPTCHA çok mu zor?\nFarklı bir soru almak için sayfayı yenileyin\nveya yanıtını %sFreenode üzerindeki #d IRC kanalında%s sorun.`;
```

Comment:
I think the correct fix for the `\n` would be to change this string to use "-delimited strings, like the English one.

Reply:
Will do.

Reply:
This has been done. @CyberShadow, do you see any other issue?

Reply:
Sorry for the delay, last few days have been a bit crowded. I will get this in ASAP.

Reply:
No worries at all. Thank you very much for all the effort you've put into the Turkish forums!